docs/modules/ROOT/pages/documentation/payara-server/notification-service/notifiers.adoc

```diff
@@ -8,6 +8,7 @@ channels apart from the standard server log mechanism:
 *JMS*:: Taking advantage of preconfigured JMS connection resources, you
```

Comment:
I'm confused, why aren't these additional notifies in their own pages? Are they going to be removed from the Community edition? If this is the case either their pages should be deleted or have a big header explained their removal

Reply:
It's explained why these are here just above. These are the *integrated* notifiers